src/components/CallView/CallView.vue

```diff
@@ -737,6 +785,39 @@ export default {
 	}
 }
 
+.viewer-overlay__video {
+	position : absolute;
+	bottom: 48px;
+	right: 8px;
+	--min-width: 130px;
+	--max-width: 180px;
+	--max-height: 180px;
+	width: 15vw;
+	height: 15vw;
+	min-width: var(--min-width);
+	max-width: var(--max-width);
```

Comment:
Not sure about px limitations: I barely see an overlay on 4K monitor. Maybe stick with 10-15vw, to always depend on screen resolution? And maybe restrict highest size to LocalVideo sizes (height: 242px)